Gems/EMotionFX/Code/EMotionFX/Tools/EMotionStudio/Plugins/StandardPlugins/Source/MotionEvents/MotionEventWidget.h

```diff
@@ -27,11 +27,12 @@ namespace EMStudio
     class MotionEventWidget
         : public QWidget
     {
-        Q_OBJECT
+        Q_OBJECT // AUTOMOC
         MCORE_MEMORYOBJECTCATEGORY(MotionEventWidget, MCore::MCORE_DEFAULT_ALIGNMENT, MEMCATEGORY_STANDARDPLUGINS);
 
     public:
-        MotionEventWidget(QWidget* parent);
+        MotionEventWidget(QWidget* parent = nullptr);
```

Comment:
nit: Might be worth adding `explicit` while we're here to do avoid any strange implicit conversions

Reply:
Changed